### ISSUE TITLE
Crashing worker

### ISSUE THREAD

**Issue by karlwessel** [fonsp/Pluto.jl#61]
When I run the following MWE notebook:
```
### A Pluto.jl notebook ###
# v0.5.11
# ╔═╡ c2fb582e-773e-11ea-00b4-3597ff4476fc
using Plots

# ╔═╡ c8f406e8-773e-11ea-36e0-39b9a7ec1f49
using Images

# ╔═╡ cb4966ec-773e-11ea-2dc9-1ff8a8ce923f
plot(Gray.(Float32[NaN 0; 0 0]))

# ╔═╡ e5635c4a-773e-11ea-34bf-9b22a8d52537


# ╔═╡ Cell order:
# ○ c2fb582e-773e-11ea-00b4-3597ff4476fc
# ○ c8f406e8-773e-11ea-36e0-39b9a7ec1f49
# ○ cb4966ec-773e-11ea-2dc9-1ff8a8ce923f
# ○ e5635c4a-773e-11ea-34bf-9b22a8d52537
```

the notebook/worker/cell crashes with the following error:
```
On worker 2:
MethodError: no method matching show(::IOContext{Base.GenericIOBuffer{Array{UInt8,1}}}, ::MIME{Symbol("text/html")}, ::InexactError)
Closest candidates are:
  show(::IO, ::MIME{Symbol("text/plain")}, ::Any) at multimedia.jl:47
  show(::IO, ::MIME{Symbol("text/html")}, ::Method) at methodshow.jl:341
  show(::IO, ::MIME{Symbol("text/html")}, ::Base.MethodList) at methodshow.jl:394
  ...
__binrepr at ./multimedia.jl:159
_textrepr at ./multimedia.jl:149
#repr#1 at ./multimedia.jl:145
inner at ./essentials.jl:715
#invokelatest#1 at ./essentials.jl:716
format_output at .julia/packages/Pluto/NMsuk/src/notebookserver/FormatOutput.jl:45
top-level scope at none:0
eval at ./boot.jl:331
#104 at 
stdlib/v1.4/Distributed/src/process_messages.jl:294
run_work_thunk at stdlib/v1.4/Distributed/src/process_messages.jl:79
macro expansion at stdlib/v1.4/Distributed/src/process_messages.jl:294 [inlined]
#103 at ./task.jl:358
Stacktrace:
 [1] remotecall_fetch(::Function, ::Distributed.Worker, ::Module, ::Vararg{Any,N} where N; kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at stdlib/v1.4/Distributed/src/remotecall.jl:390
 [2] remotecall_fetch(::Function, ::Distributed.Worker, ::Module, ::Vararg{Any,N} where N) at stdlib/v1.4/Distributed/src/remotecall.jl:382
 [3] #remotecall_fetch#144 at stdlib/v1.4/Distributed/src/remotecall.jl:417 [inlined]
 [4] remotecall_fetch at stdlib/v1.4/Distributed/src/remotecall.jl:417 [inlined]
 [5] remotecall_eval at stdlib/v1.4/Distributed/src/macros.jl:240 [inlined]
 [6] eval_fetch_in_workspace(::Pluto.WorkspaceManager.ProcessWorkspace, ::Expr) at .julia/packages/Pluto/NMsuk/src/react/WorkspaceManager.jl:167
 [7] eval_fetch_in_workspace(::Notebook, ::Expr) at .julia/packages/Pluto/NMsuk/src/react/WorkspaceManager.jl:123
 [8] run_single!(::Notebook, ::Cell) at .julia/packages/Pluto/NMsuk/src/react/Run.jl:84
 [9] run_reactive!(::Notebook, ::Array{Cell,1}) at .julia/packages/Pluto/NMsuk/src/react/Run.jl:53
 [10] macro expansion at .julia/packages/Pluto/NMsuk/src/react/Run.jl:70 [inlined]
 [11] (::Pluto.var"#63#64"{Notebook,Array{Cell,1}})() at ./task.jl:358
```

On the same note: is there already a gracefull way to recover from crashes like this? Currently I have to restart Pluto completely.

**Comment by fonsp** [comment]
Thank you for writing a MWE! The file format is proving useful already :)

You found a bug where an error during the call to `show` (after cell exectuion) is not caught properly. This is [fixed in v0.5.13](https://github.com/fonsp/Pluto.jl/commit/10301b00434fa67df26e64126feb41fc8aae59b4#diff-10adaa0ca8fcded1a3465c8662a00e9dR45) 🎈

Did you intend the cell to throw an error? The call `plot(Gray.(...))` now outputs:
```
Failed to show value: 
InexactError: trunc(UInt64, NaN)
Stacktrace:
 [1] format_output(::Plots.Plot{Plots.GRBackend}) at /mnt/c/dev/julia/Pluto.jl/src/notebookserver/FormatOutput.jl:45
 [2] top-level scope at none:0
 [3] eval(::Module, ::Any) at ./boot.jl:330
 [4] (::Distributed.var"#108#110"{Distributed.CallMsg{:call_fetch}})() at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.3/Distributed/src/process_messages.jl:294
 [5] run_work_thunk(::Distributed.var"#108#110"{Distributed.CallMsg{:call_fetch}}, ::Bool) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.3/Distributed/src/process_messages.jl:79
 [6] macro expansion at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.3/Distributed/src/process_messages.jl:294 [inlined]
 [7] (::Distributed.var"#107#109"{Distributed.CallMsg{:call_fetch},Distributed.MsgHeader,Sockets.TCPSocket})() at ./task.jl:333
```
If so, you can close this issue.

**Comment by karlwessel** [comment]
In the original code I did not intent the cell to throw an error, I just did a mistake and Pluto failed to forward the correct error message.

Now everything works like it should, thank you again!
